Sources/SwiftBuildSupport/SwiftBuildSystem.swift

```diff
@@ -276,24 +276,95 @@ public final class SwiftBuildSystem: SPMBuildCore.BuildSystem {
         self.delegate = delegate
     }
 
+    private func createREPLArguments(
```

Comment:
not sure it makes sense to keep this in the initial PR since it's more WIP/unused for now